ui/packages/app/src/project_setting/project_info/UpdateProjectInfoModal.js

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect } from "react";
+import { addToast, useMlpApi } from "@caraml-dev/ui-lib";
+import {
+  EuiConfirmModal,
+  EuiOverlayMask,
+  EuiCode,
+  EuiBasicTable,
+  EuiFlexItem,
+  EuiFlexGroup
+} from "@elastic/eui";
+import { useNavigate } from "react-router-dom";
+
+const UpdateProjectInfoModal = ({
+  project,
+  closeModal,
+  fetchUpdates,
+  originalProject
+}) => {
+  const [submissionResponse, submitForm] = useMlpApi(
+    `/v1/projects/${project.id}`,
+    {
+      method: "PUT",
+      headers: { "Content-Type": "application/json" }
+    },
+    {},
+    false
+  );
+
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (submissionResponse.isLoaded && !submissionResponse.error) {
+      closeModal();
+      addToast({
+        id: "submit-success-create",
+        title: "Project Info Updated!",
+        color: "success",
+        iconType: "check"
+      });
+      fetchUpdates();
+      navigate(`/projects/${submissionResponse.data.id}/settings/project-info`);
+    }
+  }, [navigate, submissionResponse, project, fetchUpdates, closeModal]);
+
+  const handleUpdate = () => {
+    const updatedProjectInfo = {
+      ...project,
+      stream: project.stream,
+      team: project.team,
+      labels: project.labels
+    };
+
+    submitForm({ body: JSON.stringify(updatedProjectInfo) });
+  };
+
+  const columnsNew = [
+    {
+      field: "key",
+      name: "Key"
+    },
+    {
+      field: "value",
+      name: "Value"
+    }
+  ];
```

Comment:
There's `columnsNew` and `columnsCurrent` - are these expected to be different? If they are going to always be the same can we just have 1 definition and reuse it?